### PR TITLE
Move accepted-claims-set cddl to new file (issue 97)

### PR DESCRIPTION
**Pull request by andrew-draper**

### CODE REVIEW

File: cddl/accepted-claims-set.cddl [hunk 1]
```diff
@@ -0,0 +1,6 @@
+accepted-claims-set = {
+  &(evidence-triples: 0) => [ + reference-triple-record ]
+  ? &(identity-triples: 2) => [ + identity-triple-record ]
+  ? &(coswid-triples: 6) => [ + ev-coswid-triple-record ]
```
Comment (thomas-fossati) [inline cddl/accepted-claims-set.cddl:4]: There is no reason for the indexes to mirror CoMID's: this is a completely separate namespace.  I suggest we move to 0, 1, 2, ... instead.

**Review by thomas-fossati — COMMENTED** [review]
LGTM

I have left a minor comment inline.

File: cddl/accepted-claims-set.cddl [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+  * $$triples-map-extension
```
Comment (thomas-fossati) [inline cddl/accepted-claims-set.cddl:5]: conventionally this should be called `accepted-claims-set-extension`

File: cddl/accepted-claims-set.cddl (hunk 1 above)
Comment (andrew-draper) [reply cddl/accepted-claims-set.cddl:4]: ok, done

File: cddl/accepted-claims-set.cddl (hunk 2 above)
Comment (andrew-draper) [reply cddl/accepted-claims-set.cddl:5]: ok

**Review by nedmsmith — APPROVED** [review]

**Review by nedmsmith — APPROVED** [review]

**Review by thomas-fossati — APPROVED** [review]
LGTM, thanks

**Pull request merged**